Enable bandwidth-manager by default for new deployments

qmonnet opened the pull request:
For Cilium v1.10, bandwidth manager is no longer considered as a beta feature: re-enable it by default for new deployments.

This PR reverts the commit that disabled the feature by default, but it does not update the documentation that still marks the bandwidth manager as _beta_.

Reverts part of https://github.com/cilium/cilium/pull/13515.
Also related: Follow-ups for EDT https://github.com/cilium/cilium/issues/13371.

qmonnet committed:
Revert "EDT: disable bandwidth-manager for new deployments"

Bandwidth manager was a beta feature in Cilium 1.9, and as such we
disabled it by default for new deployments. For newer versions, enable
bandwidth manager by default.

This reverts commit a0ad5ad47f60947c409aae4f53afb9c58c566bbd.

Signed-off-by: Quentin Monnet <quentin@isovalent.com>

christarazi commented on install/kubernetes/cilium/values.yaml:
```diff
@@ -173,7 +173,7 @@ azure:
 
 # TODO: Add documentation
-bandwidthManager: false
+bandwidthManager: true
```
Would this be the time to add documentation?

qmonnet replied:
Why not, this commit is a revert but I can add a new one to comment the value. Thanks!

qmonnet committed:
install/kubernetes: add documentation for "bandwidthManager" value

Add a comment to implement the TODO and provide some documentation for
the "bandwidthManager" value in the yaml file.

Signed-off-by: Quentin Monnet <quentin@isovalent.com>
